leshan-server-core/src/main/java/org/eclipse/leshan/server/bootstrap/BootstrapConfig.java

```diff
@@ -465,6 +456,56 @@ public String toString() {
         }
     }
 
+    public class CipherSuiteId {
+
+        private final byte firstByte;
+        private final byte secondByte;
+
+        /**
+         * Ciphersuite is created with 2 bytes. Possible values are described
+         * in the <a href="https://iana.org/assignments/tls-parameters/tls-parameters.xhtml#tls-parameters-4">
+         *     registry</a>.
+         *
+         * @param firstByte first byte of ciphersuite (for example 0xC0)
+         * @param secondByte second byte of ciphersuite (for example 0xA8)
+         */
+        public CipherSuiteId(byte firstByte, byte secondByte) {
+            this.firstByte = firstByte;
+            this.secondByte = secondByte;
+        }
+
+        /**
+         * Integer is split into 2 bytes for example 49320 (0xc0a8 in hex) will be split into "0xC0,0xA8".
+         * This format is used by Security Object, resource 16.
+         *
+         * @param valueFromSecurityObject Integer representing ciphersuite id
+         */
+        public CipherSuiteId(ULong valueFromSecurityObject) {
+            String binaryString = Long.toBinaryString(valueFromSecurityObject.longValue());
+            this.firstByte = (byte) Integer.parseInt(binaryString.substring(0, 8), 2);
+            this.secondByte = (byte) Integer.parseInt(binaryString.substring(9, 17), 2);
```

Comment:
Using String is maybe not the best idea.

Should be better to play with binary operator like : 
- shift operator :  `>>`

You can create unit test to check your code.

We should also check that `valueFromSecurityObject` is a 16-bit unsigned integer.